### PR TITLE
query backends from installed packages

### PR DESCRIPTION
**Pull request by edoput**
Hi all,

this will solve a problem that I had last summer during GSOC. Using `entry_points` in the definition
of `setup.py` we can query possible backends for use in netjsonconfig in other packages instead of having long-lived branches.

This is an example for a `setup.py` using the `entry_points` directive

```python
from setuptools import setup, find_packages                                                                                                           
                                                                                                                                                      
setup(                                                                                                                                                
    name='airos',                                                                                                                                  
    version='0.0.0',                                                                                                                                  
    description='AirOS backend for netjsonconfig',                                                                                                                          
    packages=find_packages(),                                                                                                                         
    entry_points={                                                                                                                                    
        'netjsonconfig.backends': [                                                                                                                   
            'airos=netjsonconfig-airos.backend:AirOsBackend',                                                                                                    
        ]                                                                                                                                             
    }                                                                                                                                                 
) 
```

### CODE REVIEW

**Comment by redhog** [comment]
I greatly support this patch as it's The Right Way To Do it<tm> :P Please merge :)

File: netjsonconfig/__init__.py [hunk 1]
```diff
@@ -15,5 +16,8 @@ def get_backends():
     }
 
     for entry_point in iter_entry_points('netjsonconfig.backends'):
-        default.update({entry_point.name.lower(): entry_point.load()})
+        try:
+            default.update({entry_point.name.lower(): entry_point.load()})
+        except ImportError as e:
+            print("Error loading backend {}".format(entry_point.name.lower()), file=sys.stderr)
```
Comment (redhog) [inline netjsonconfig/__init__.py]: Shouldn't this rather use the python logging framework?

Comment (redhog) [reply netjsonconfig/__init__.py]: [django-netjsonconfig](https://github.com/openwisp/django-netjsonconfig/blob/03a9ef8f282e6de57e939cc47733230cbfdcf1b8/django_netjsonconfig/utils.py#L9] uses

    import logging
    logger = logging.getLogger(__name__)
    logger.warning("Grrrrrr")

Comment (edoput) [reply netjsonconfig/__init__.py]: Ok I'll change this to be a logger on the console :+1:

**Comment by edoput** [comment]
Let me know if there's anything else to do or I'll squash everything before merge

**Pull request merged**